PluginSetup.php

```diff
@@ -37,5 +39,21 @@ public static function maybe_deactivate() {
 			wp_die();
 			exit;
 		}
+
+		if ( ! is_plugin_active( self::WOOCOMMERCE_PATH ) ) {
```

Comment:
Can we create a  function in order to don't DRY in both places where the plugin should get deactivated.